### ISSUE TITLE
Carat movement past EOL

### ISSUE THREAD

**Issue by carlca** [Microsoft/vscode#37007]
- VSCode Version: VSCode 1.18 Insider
- OS Version: macOS 10.13.0 High Sierra

Steps to Reproduce:

1. Open file to edit - I tend to edit .go files
2. Place carat on any line, and press cursor down.
3. I would like the ability to move the cursor down regardless of line length, entering the space beyond end-of-line if necessary.
4. Delphi has always allowed this, and I think, as an option, it would improve the perceived speed of the editor

N/A

**Comment by mike-mo** [comment]
Is this the same as #13960 then?

**Comment by carlca** [comment]
Yes it is. I had forgotten the term “virtual space”! In fact I’ve since seen a couple of Github issue on the subject, including one which suggested that it unlikely to be addressed in the foreseeable future. Which is a shame I think. I think that its implementation would greatly add to the already good impression of a speedy and responsive UI.

Cheers,
Carl

**Comment by AnrDaemon** [comment]
Just a reminder that this is a duplicate of #13960
